### ISSUE TITLE
mimemagic dependency is broken

### ISSUE THREAD

**Issue by thill-odi** [innoq/iqvoc#415]
I've attempted to deploy to Heroku using the instructions given in the README and get the following error:
```
Your bundle is locked to mimemagic (0.3.5), but that version could not be found
in any of the sources listed in your Gemfile. If you haven't changed sources,
that means the author of mimemagic (0.3.5) has removed it. You'll need to update
your bundle to a version other than mimemagic (0.3.5) that hasn't been removed
in order to install.
```
I assume this is connected to the issues described at e.g. https://stackoverflow.com/questions/66844316/bundlergemnotfound-could-not-find-mimemagic-0-3-5-in-any-of-the-sources-on-ra

I've attempted to fix the dependencies myself, but this has been fruitless and led to other dependency issues.

**Comment by mjansing** [comment]
Thanks for the report. I'll take a look at it.

**Comment by mjansing** [comment]
Should be fixed with 5b45a201c404910bdab417640ebcbe414159f226.
